roles/perfmon_configure/tasks/configure.yml

```diff
@@ -88,7 +88,7 @@
 - name: configure | Check before enable nodes for performance collection #TODO: Only checks first node for perfmon.
   vars:
     sensor_nodes: "{{ ansible_play_hosts | list }}"
-  shell: "/usr/lpp/mmfs/bin/mmlscluster -Y | grep -v HEADER | grep clusterNode |grep {{ sensor_nodes | first }} | cut -d ':' -f 14"
+  shell: "/usr/lpp/mmfs/bin/mmlscluster -Y | grep -v HEADER | grep clusterNode | grep {{ sensor_nodes | map('extract', hostvars, 'scale_daemon_nodename') | first }} | cut -d ':' -f 14"
```

Comment:
@acch I think now `scale_daemon_nodename`  will refer to inventory hostname correct ?

Reply:
@rajan-mis No. `sensor_nodes` contains inventory hostname(s), which could really be any identifier. `mmlscluster` on the other hand requires the `scale_daemon_nodename` (defaults to `ansible_hostname` but could be overridden). Hence, this added code "translates" the inventory hostname into the daemon nodename... it extracts the variable for a given inventory item.

Does that make sense?

Reply:
Yes sure. Thanks @acch